fedimintd/src/bin/main.rs

```diff
@@ -83,22 +84,28 @@ async fn main() -> anyhow::Result<()> {
     let db: Database = fedimint_rocksdb::RocksDb::open(opts.cfg_path.join(DB_FILE))
         .expect("Error opening DB")
         .into();
-    let btc_rpc = fedimint_bitcoind::bitcoincore_rpc::make_bitcoind_rpc(&cfg.wallet.btc_rpc)?;
+    let btc_rpc = fedimint_bitcoind::bitcoincore_rpc::make_bitcoind_rpc(
+        &cfg.get_module_config::<WalletConfig>("wallet")?.btc_rpc,
+    )?;
```

Comment:
This tells me that the wallet module should connect to bitcoind in its constructor. Just a future refactor. We might want a common constructor interface for modules.